### PR TITLE
Update 'Drupal Performance and Varnish Caching Settings' page

### PR DESCRIPTION
**Pull request by greg-1-anderson**
Include information about Drupal 8 cache settings.

Closes #n/a

## Effect
PR includes the following changes:
- Adds a "Drupal 8" section to Drupal Performance and Varnish Caching Settings

## Remaining Work
- [x] Technical review
- [x] Documentation / English review
- [x] Wait until Drupal 8.3.3 is deployed to publish (enables behavior described here)
- [x] Consider whether to add another # to the headings in the Drupal 7 section while we're here; seems that they are all one size too large.

### CODE REVIEW

**Comment by rachelwhitton** [comment]
+1

File: source/_docs/drupal-cache.md [hunk 1]
```diff
@@ -6,8 +6,25 @@ categories: [drupal]
...
 
+Visit `/admin/config/development/performance` for Drupal's performance settings.
+
+### Caching
+![Drupal 8 Caching options](/source/docs/assets/images/d8-cache-config.png)  
+**This is a key setting**. It determines what value Drupal delivers in its `max-age` header, which is how long the reverse-proxy layer will retain a cache.
+
+Performance is often a trade-off between how fresh your content is, and how fast you want to deliver it to the internet. A good value to start with is 15 minutes, but this is something to consider. If you can set it to an hour, that's great for performance. More than a day is usually excessive, since the edge cache will decay over that amount of time in most cases.
+
+On Pantheon, this value defaults to 15 minutes. This is done on the first cache-clear operation on the site; immediately after installing the site, you may see this set to `&lt;no caching&gt;`. In this case, press the "Clear all caches" button, or select the page cache maximum age from the available selections.
```
Comment (rachelwhitton) [inline source/_docs/drupal-cache.md]: Character entities wrapped in code styling are not converted to characters, suggest `<no caching>` instead.

**Comment by rachelwhitton** [comment]
Recommend we delete [/docs/drupal-8-cache/](https://pantheon.io/docs/drupal-8-cache) and redirect.

**Comment by greg-1-anderson** [comment]
The code behind this has been deployed, so this can be published when ready.

Replacing /docs/drupal-8-cache/ with this content seems like a good idea. There is some content in the old page (e.g. link to W3C spec) that should be merged in here.

**Pull request merged**